### PR TITLE
Use H2 domain for comments. Bump the version to match the JS

### PR DESCRIPTION
**Pull request by richard-still-ft**

### CODE REVIEW

File: client/main.scss [hunk 1]
```diff
@@ -15,8 +15,8 @@ $o-gallery-is-silent: false;
 // of o-comments/o-comment-ui will appear on the site.
 @import "o-assets/main";
 @include oAssetsSetModulePaths((
-	o-comments: '//build.origami.ft.com/files/o-comments@2.0.3',
-	o-comment-ui: '//build.origami.ft.com/files/o-comment-ui@2.0.2'
+	o-comments: '//next-geebee.ft.com/build/files/o-comments@2.0.5',
```
Comment (matthew-andrews) [inline client/main.scss]: 6?
https://github.com/Financial-Times/next-article/blob/use-h2-domain-for-comments/bower.json#L14

**Comment by ironsidevsquincy** [comment]
ooooooollllllllllllldddddd

**Comment by imranolas** [comment]
This PR is very old and unmergeable.
@richard-still-ft is it safe for me close this?

**Comment by imranolas** [comment]
I'm closing this for the moment but I'll keep the branch open.

**Pull request closed without merging**